librz/bin/format/elf/elf_info.c

```diff
@@ -433,6 +433,25 @@ static ut64 get_main_offset_mips(ELFOBJ *bin, ut64 entry, ut8 *buf, size_t size)
 	return 0;
 }
 
+static ut64 get_main_offset_v850(ELFOBJ *bin, ut64 entry, ut8 *buf) {
```

Comment:
can you provide the disassembly of what these instructions are in a comment? So that we can understand what this function is trying to recognize